pkg/git/branchWalker.go

```diff
@@ -232,3 +234,13 @@ func (b *branchWalker) reconcileCommit(hash string, version *gitVersion) error {
 
 	return nil
 }
+
+func parseTag(tag string) (*semver.Version, error) { // TODO: Support ignoring invalid semver tags
+	trimmedTag := strings.TrimPrefix(tag, "v")
+	version, err := semver.NewVersion(trimmedTag)
+	if err != nil {
+		return nil, err
+	}
+
+	return version, nil
+}
```

Comment:
weird that go fmt didn't add the newline here

Reply:
Just realized that my VSCode formatter was using Prettier instead of fmt. All better now